app/services/email_style_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class EmailStyleUpdater
+
+  attr_reader :errors
```

Comment:
This might be a good fit for the `HasErrors` mixin which includes a way to add errors easily, and the serializer knows how to render it for display in a popup.